lib/Github/Client.php

```diff
@@ -73,13 +73,17 @@ class Client
      * Authenticate using a client_id/client_secret combination.
      *
      * @var string
+     *
+     * @deprecated Use the AuthMethod const
```

Comment:
```suggestion
     * @deprecated Use the Github\AuthMethod::CLIENT_ID const
```

Directly suggest the corresponding constant to make it easier for users who need to change their code